_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,190 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope), except if the element has an implicit semantic role that is identical to its explicit semantic role.
```

Comment:
From conversation with @Brynanders: 
At least add an inapplicable test case for this.

Reply:
@WilcoFiers, I think the original intent was to keep html validation stuff out of this rule. 
However, I guess it would be more in line with the other rules if we include it...

Reply:
@WilcoFiers, correction to my previous comment: 

As I see it, there are two options: 
- keep it as it is, making this case inapplicable: `<li role="listitem">`, OR
- make it apply to "any HTML or SVG element that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has a [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope).", making both these cases applicable: `<li role="listitem">` and `<li>`.

Reply:
My vote is for option one than.

Reply:
Inapplicable test case added for case where explicit role is same as implicit role.